__tests__/helpers/create-user.js

```diff
@@ -0,0 +1,43 @@
+const { gql } = require('@apollo/client');
+const client = require('../../utils/apollo-test-client');
+
+module.exports = async ({ firstName, lastName, email, password, language }) => {
```

Comment:
Please give this arrow function name and export object with this variable.

Reply:
Done